src/app/pages/swap/components/swap-details/stacks-swap-details.tsx

```diff
@@ -0,0 +1,93 @@
+import { SwapSelectors } from '@tests/selectors/swap.selectors';
+import BigNumber from 'bignumber.js';
+import { useFormikContext } from 'formik';
+import { HStack, styled } from 'leather-styles/jsx';
+
+import type { SwapAsset } from '@leather.io/query';
+import { ChevronRightIcon } from '@leather.io/ui';
+import { createMoneyFromDecimal, formatMoney, isDefined, isUndefined } from '@leather.io/utils';
+
+import type { SwapFormValues } from '@shared/models/form.model';
+
+import { useSwapContext } from '@app/pages/swap/swap.context';
+
+import type { StacksSwapContext } from '../../providers/stacks-swap-provider';
+import { toCommaSeparatedWithAnd } from '../../swap.utils';
+import { SwapDetailLayout } from './swap-detail.layout';
+import { SwapDetailsLayout } from './swap-details.layout';
+import { getStacksSwapDataFromRouteQuote } from './swap-details.utils';
+
+function RouteNames(props: { router?: SwapAsset[] }) {
```

Comment:
Another name/greater qualification might be better here, I thought this was to do with the router, not the route that the funds take across dexes